src/Controller/ProfileController.php

```diff
@@ -24,6 +24,7 @@ class ProfileController
      *         @OA\Property(
      *             type="object",
      *             property="data",
+     *             description="Data",
```

Comment:
Data can also be an empty string, what matters is to have the `description` field
```suggestion
     *             description="",
```